spec/models/importers/organisation_spec.rb

```diff
@@ -31,29 +31,22 @@
   end
 
   context 'Content Items' do
-    it 'imports all content items for the organisation' do
+    before do
       allow(HTTParty).to receive(:get).and_return(two_content_items_response)
       Importers::Organisation.new('a-slug').run
-      organisation = Organisation.find_by(slug: 'a-slug')
-
-      expect(organisation.content_items.count).to eq(2)
+      @organisation = Organisation.find_by(slug: 'a-slug')
```

Comment:
1. Duplication in the code that exercises the code is accepted if it increases readability. I think this is the case
2. It is also easier to add extra setup code for the test (if we use a before block we can't do it per test)

I would go with something like this:

```ruby
before { allow(HTTParty).to receive(:get).and_return(two_content_items_response) } 
let(:organisation) { organisation = Organisation.find_by(slug: 'a-slug') }

it 'imports a `content_id` for every content item' do
  Importers::Organisation.new('a-slug').run

  content_ids = organisation.content_items.pluck(:content_id)
  expect(content_ids).to eq(%w(content-id-1 content-id-2))
end
```